src/viewer/mobile/Panel.tsx

```diff
@@ -264,6 +264,7 @@ class Panel extends React.Component<Props & TwitchExtProps, State> {
 
 	private onCopy = () => {
 		this.setState({ copied: true });
+		ga("send", "event", "Click", "Copy Deck", "Mobile");
```

Comment:
You will also need to initialize GA in `mobile/index.tsx`. If you follow my recommendation above you can just include the `analytics.ts` file there.